python/rmm/rmm/_lib/memory_resource.pyx

```diff
@@ -1188,3 +1188,10 @@ def get_log_filenames():
         else None
         for i, each_mr in _per_device_mrs.items()
     }
+
+
+def get_free_device_memory():
+    """
+    Return the number of bytes of available device memory.
+    """
+    return percent_of_free_device_memory(percent)
```

Comment:
Where is `percent` defined? Is this leftover and still needs an update?

Reply:
Sorry, this was a left-over. Fixed it.